html/toDoLists.html

```diff
@@ -34,7 +34,7 @@ <h2 class="appMoto">Be active, be nice</h2>
 
 
 <section class="plus">
-<a href="" id="addTask"><i class="fas fa-plus"></i></i></a>
+<a href="#" id="addTask"><i class="fas fa-plus"></i></i></a>
```

Comment:
can you explain why you did that?

Reply:
Because every time we press the add-on button, it reloads the page itself, and this affects Java text, so Adnan stopped reloading by tagging #in the tag anchor.